containers/bash-scheduled-job/serverless.yml

```
@@ -0,0 +1,22 @@
+service: scaleway-container
+
+provider:
+  name: scaleway
+
+plugins:
+  - serverless-scaleway-functions
+
+package:
+  patterns:
+    - '!node_modules/**'
+    - '!.gitignore'
+    - '!.git/**'
+    - '!.idea/**'
+
+custom:
+  containers:
+    first:
```

Could you rename it to something more explicit like:
```suggestion
    scheduled-bash-job:
```